thorlcr/activities/hashdistrib/thhashdistribslave.cpp

```diff
@@ -879,20 +879,22 @@ class CDistributorBase : implements IHashDistributor, implements IExceptionHandl
                     unsigned dest=iter->get();
                     CTarget *target = targets.item(dest);
                     Owned<CSendBucket> bucket = target->getBucketClear();
-                    HDSendPrintLog3("Looking at last bucket: target=%s, size=%u", target->queryInfo(), bucket.get()?bucket->querySize():-1);
                     if (bucket && bucket->querySize())
                     {
-                        HDSendPrintLog3("Sending last bucket(s): target=%s, size=%u", target->queryInfo(), bucket->querySize());
+                        //HDSendPrintLog3("Sending last bucket(s): target=%s, size=%u", target->queryInfo(), bucket->querySize());
```

Comment:
not output in release anyway, but yes too verbose in debug build.
I'd be content with these being removed altogether, they haven't been useful in a long time.